testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -220,6 +219,16 @@ def test_load_new(self):
         u.load_new(PDB_small)
         assert_equal(len(u.trajectory), 1, "Failed to load_new(PDB)")
 
+    def test_load_new_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(PDB_small)
+        assert result is u
+
+    def test_load_new_None_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(None)
```

Comment:
Sure but it makes more sense to remove the `self.load_new(None)` call in `__init__` rather than have the weird behaviour and associated test